test/integration/pipeline/transfer_asset_inter_domain_test.cpp

```diff
@@ -28,7 +28,7 @@ class TransferAssetInterDomainTest : public TxPipelineIntegrationTestFixture {
 
     // creates node and admin keys and generate default genesis transaction
     auto genesis_tx1 =
-        TransactionGenerator().generateGenesisTransaction(0, {"0.0.0.0:10001"});
+        TransactionGenerator().generateGenesisTransaction(0, {"0.0.0.0:50541"});
```

Comment:
You are using this number twice in the same file, maybe it will be better to declare a constant?